src/app/app-modules/lab/workarea/workarea.component.html

```
@@ -1,747 +1,589 @@
-<div class="container-fluid" *ngIf="technicianForm">
-  <mat-sidenav-container class="example-sidenav-fab-container">
-    <mat-sidenav #sidenav mode="side" opened="false">
-      <div class="example-scrolling-content">
-        <app-beneficiary-details></app-beneficiary-details>
-      </div>
-    </mat-sidenav>
+<div class="relative px-4 py-6" *ngIf="technicianForm">
```

HTMLHint (1.9.2)

[error] Doctype must be declared before any non-comment content.

(doctype-first)